src/utils/validateIdentification.ts

```diff
@@ -0,0 +1,27 @@
+import { validateAndCleanNric } from "./validateNric";
+import { validateAndCleanRegexInput } from "./validateInputWithRegex";
+import { EnvVersionError } from "../services/envVersion";
+
+export const validateAndCleanId = (
+  inputId: string,
+  idValidation: string | undefined,
+  idRegex: string | undefined
```

Comment:
Shall we set these as optional? So if consumers call `validateAndCleanId("ASD")` it works fine without needing the 2nd and 3rd args

Reply:
eh i think i just added this liao 😂😂

Reply:
oh haha, how about `idValidation` too?

Reply:
OH oops missed that, adding it in, thanks!🙏